sencha/constants_translate.py

```diff
@@ -52,6 +52,8 @@
     ],
 )
 
+SingleFileScore = namedtuple("SingleFileScore", ["scoring_lines", "fasta_seqs"],)
```

Comment:
i really like using a named tuple for this! Makes things easier to understand/debug